macros/edr/data_monitoring/monitors_query/table_monitoring_query.sql

```diff
@@ -1,17 +1,18 @@
-{% macro table_monitoring_query(monitored_table_relation, timestamp_column, is_timestamp, min_bucket_start, table_monitors, freshness_column=none) %}
+{% macro table_monitoring_query(monitored_table_relation, timestamp_column, min_bucket_start, table_monitors, freshness_column=none, where=none) %}
 
     {%- set max_bucket_end = "'"~ elementary.get_run_started_at().strftime("%Y-%m-%d 00:00:00")~"'" %}
     {%- set max_bucket_start = "'"~ (elementary.get_run_started_at() - modules.datetime.timedelta(1)).strftime("%Y-%m-%d 00:00:00")~"'" %}
     {% set full_table_name_str = "'"~ elementary.relation_to_full_name(monitored_table_relation) ~"'" %}
 
-    {% if is_timestamp %}
+    {% if timestamp_column %}
         with filtered_monitored_table as (
             select *,
                    {{ elementary.time_trunc('day', timestamp_column) }} as start_bucket_in_data
             from {{ monitored_table_relation }}
             where
                 {{ elementary.cast_as_timestamp(timestamp_column) }} >= {{ elementary.cast_as_timestamp(min_bucket_start) }}
                 and {{ elementary.cast_as_timestamp(timestamp_column) }} <= {{ elementary.cast_as_timestamp(max_bucket_end) }}
+                {% if where %} and {{ where }} {% endif %}
```

Comment:
Should we add the `where` to the other tests as well or stick with `table_anomalies`, `volume`, `freshness` for now?